[2024-08-30T17:39:20.463+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_Dag_S3.extract_hr scheduled__2024-08-30T16:00:00+00:00 [queued]>
[2024-08-30T17:39:20.477+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_Dag_S3.extract_hr scheduled__2024-08-30T16:00:00+00:00 [queued]>
[2024-08-30T17:39:20.478+0000] {taskinstance.py:2170} INFO - Starting attempt 4 of 9
[2024-08-30T17:39:20.504+0000] {taskinstance.py:2191} INFO - Executing <Task(SqlToS3Operator): extract_hr> on 2024-08-30 16:00:00+00:00
[2024-08-30T17:39:20.522+0000] {standard_task_runner.py:60} INFO - Started process 271 to run task
[2024-08-30T17:39:20.536+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'ETL_Dag_S3', 'extract_hr', 'scheduled__2024-08-30T16:00:00+00:00', '--job-id', '119', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag_s3.py', '--cfg-path', '/tmp/tmp7zzn4_ju']
[2024-08-30T17:39:20.543+0000] {standard_task_runner.py:88} INFO - Job 119: Subtask extract_hr
[2024-08-30T17:39:20.700+0000] {task_command.py:423} INFO - Running <TaskInstance: ETL_Dag_S3.extract_hr scheduled__2024-08-30T16:00:00+00:00 [running]> on host 00c82e2512e7
[2024-08-30T17:39:20.928+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='jasontran' AIRFLOW_CTX_DAG_ID='ETL_Dag_S3' AIRFLOW_CTX_TASK_ID='extract_hr' AIRFLOW_CTX_EXECUTION_DATE='2024-08-30T16:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-30T16:00:00+00:00'
[2024-08-30T17:39:20.952+0000] {base.py:83} INFO - Using connection ID 'PostgreSQL_conn' for task execution.
[2024-08-30T17:39:20.979+0000] {base.py:83} INFO - Using connection ID 'PostgreSQL_conn' for task execution.
[2024-08-30T17:39:21.094+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/providers/common/sql/hooks/sql.py:221 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2024-08-30T17:39:21.179+0000] {sql_to_s3.py:179} INFO - Data from SQL obtained
[2024-08-30T17:39:21.186+0000] {sql_to_s3.py:186} INFO - Writing data to temp file
[2024-08-30T17:39:21.224+0000] {sql_to_s3.py:189} INFO - Uploading data to S3
[2024-08-30T17:39:21.239+0000] {base.py:83} INFO - Using connection ID 'AWS_S3_conn' for task execution.
[2024-08-30T17:39:21.241+0000] {connection_wrapper.py:381} INFO - AWS Connection (conn_id='AWS_S3_conn', conn_type='aws') credentials retrieved from extra.
[2024-08-30T17:39:26.519+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=ETL_Dag_S3, task_id=extract_hr, execution_date=20240830T160000, start_date=20240830T173920, end_date=20240830T173926
[2024-08-30T17:39:26.632+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-08-30T17:39:26.714+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
